index.html

```diff
@@ -1389,14 +1389,24 @@ <h3>HTML Element Role Mappings</h3>
               <th>
                 <a data-cite="HTML">`img`</a>
               </th>
-              <td class="aria"><a class="core-mapping" href="#role-map-img">`img`</a> role</td>
+              <td class="aria">
+                <a class="core-mapping" href="#role-map-img">`img`</a> role
+                or
+                <!-- todo: update role map link if/when core aam adds in a specific image role mapping table -->
+                <a class="core-mapping" href="#role-map-img">`image`</a>
```

Comment:
```suggestion
                <a class="core-mapping" href="#role-map-image">`image`</a>
```

As I said before, I do think there should be (as there is now) an entry for every role in CORE-AAM. And this link should go to "image" role as it suggests.

Reply:
then this PR is blocked on core aam not having an image role entry

Reply:
or we can merge as is, and then update the link once the image role is added to core aam.

i prefer doing that for something so minor, like this.  but will wait if others disagree

Reply:
omg! I was confused by your comment because I was so convinced there was an entry for "image". Tomorrow I'm going to merge the changes related to removing the tables/disclosure, I'll also open a PR adding image, and then you can merge this, how about that?